src/ChatWindow.test.js

```diff
@@ -113,7 +113,47 @@ it("handles gracefully when pushMessage is called with an empty or null message"
   expect(finalState).toEqual(originalState);
 });
 
+test("that when a user submits the form, we call AVSGateway with their request even if the access_token is not available.", () => {
+  const authInfoWithMissingAccessToken = { access_token: undefined };
```

Comment:
What is there is no access_token prop?

Reply:
Added an additional object to test setup that doesn't contain access_token.